### PR TITLE
Improve documentation for "Vehicle Geometry with OpenFOAM" app

### PR DESCRIPTION
**Pull request by nathanrooy**
Issue for app: #[issue number here]

# App pull request
- [ ] This is a new app
- [x] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version):
- Current gallery app URL: https://dash-gallery.plotly.host/dash-vehicle-geometry/

## Workflow

- [ ] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [ ] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [ ] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [ ] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [ ] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [ ] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [ ] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [ ] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

## Post PR (at merge time)

- [ ] When you are merging, make sure to write one of the following tags in the commit message (or it will default to patch):
  - `#patch` - An app has been updated or fixed
  - `#minor` - A new app has been added, or an app has been significantly reworked
  - `#major` - Breaking changes, make sure to discuss with dash-core before using this tag

### CODE REVIEW

**Comment by xhlulu** [comment]
@nathanrooy can you keep `requirements.txt` blank (no requirements) and `requirements-real.txt` with the real (frozen) requirements? This is because the dependencies are installed during predeploy instead of at initialization so i had to use a file with a different name.

File: apps/dash-vehicle-geometry/README.md [hunk 1]
```diff
@@ -1 +1,28 @@
-# plotly-vtk
\ No newline at end of file
+# Dash: Vehicle Geometry with OpenFOAM
+
+by Nathan Rooy
```
Comment (xhlulu) [inline apps/dash-vehicle-geometry/README.md:3]:
```suggestion
by [Nathan Rooy](https://github.com/nathanrooy)
```

File: apps/dash-vehicle-geometry/README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+```
+pip install -r requirements.txt
+```
+
+Next, run `app.py` which will launch a local Dash server:
+```
+python app.py
+```
+
+## References
```
Comment (xhlulu) [inline apps/dash-vehicle-geometry/README.md:24]:
```suggestion
## References
* Original repository: https://github.com/nathanrooy/dash-vtk-cfd
```

**Review by xhlulu — CHANGES_REQUESTED** [review]

**Comment by nathanrooy** [comment]
Hi @xhlulu, just reverted the `requirements.txt` and the `requirements-real.txt` per your request.

**Review by xhlulu — APPROVED** [review]

**Pull request merged**